packages/flutter/lib/src/material/dialog.dart

```diff
@@ -1428,6 +1428,7 @@ Future<T?> showDialog<T>({
   RouteSettings? routeSettings,
   Offset? anchorPoint,
   TraversalEdgeBehavior? traversalEdgeBehavior,
+  Duration transitionDuration = const Duration(milliseconds: 150),
```

Comment:
The `Material` widget has a `static const defaultSplashRadius` member:

https://github.com/flutter/flutter/blob/3bda455f4ef6fd626c66a986b69b512f412f7bab/packages/flutter/lib/src/material/material.dart#L439-L440

I think it'd be nice for `DialogRoute` and `CupertinoDialogRoute` to have something similar that stores the default duration—a single source of truth will make it easier to tweak in the future if needed.

Reply:
i added, thank you.